androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -54,4 +85,20 @@ fun EnhancedVideoPlayer(
             }
         }
     )
+
+    if (useControls) {
+        PlayerControls(
+            isPlaying = isPlaying,
+            onPreviousClick = { exoPlayer.seekToPrevious() },
+            onNextClick = { exoPlayer.seekToNext() },
+            onPauseToggle = {
+                if (isPlaying) {
+                    exoPlayer.pause()
+                } else {
+                    exoPlayer.play()
+                }
+            },
```

Comment:
Those can be defined as default values to those callback arguments. We are probably going to pass the `exoPlayer` reference anyway if we want to customize inside the `PlayerControls` overlay